Jenkinsfile

```diff
@@ -0,0 +1,18 @@
+node("docker && awsaccess") {
+  cleanWs()
+  checkout scm
+
+  stage("Build Image") {
```

Comment:
if you don't plan to add more testing or publish the test reports you can reduce all this to 

```
node("docker && awsaccess") {
  cleanWs()
  checkout scm

  stage("Unit tests") {
     sh "docker-compose run --rm gem-tests"
  }
}
```